.github/CODEOWNERS

```
@@ -0,0 +1,3 @@
+# automatically requests pull request reviews for files matching the given pattern; the last match takes precendence
+
+*       @spacetelescope/rad-maintainers 
```

```suggestion
*       @spacetelescope/rad-maintainers @WilliamJamieson
```

@WilliamJamieson also requested to get notifications to keep up to date with schema changes